placeholders.go

```diff
@@ -46,6 +47,18 @@ func parsePlaceholders(input string, r *http.Request) string {
 			}
 			input = strings.Replace(input, "{user}", user, -1)
 		}
+		if strings.HasPrefix(placeholder[0], "{label") {
+			nStr := placeholder[0][6 : len(placeholder[0])-1] // get the integer N in "{labelN}"
+			n, err := strconv.Atoi(nStr)
+			if err != nil || n < 1 {
+				input = strings.Replace(input, placeholder[0], "", -1)
```

Comment:
Why the decision to not error here? {label0} is not supported so we could just fail or?

Reply:
In the caddy implementation of labelN, an empty string is returned instead of failing, so my thinking was that we could do that here. Would it be better to `log.Fatal()` here?

Reply:
"Someone else did" shouldn't always be the base for a decision. Think through the actual use and failure cases. Why is a decision good or not. Don't just copy and paste. Think about what the actual solution should be doing from both a user perspective and a code perspective.

Reply:
So let's elaborate what the different failure cases are.
When would this hit?
When would a user do this?
Should we allow a value/placeholder that never actually does anything?
Should we diverge from caddy's implementation and actually make `{label0}` aka the top level tld work?
etc.

Reply:
The main point in our usage is probably that we have fallback code that would redirect on error.
So with it being replaced with an empty string we won't trigger the fallback code, but maybe we should.

Reply:
So an error would occur if the `N` in a label can't be converted to an int by `strconv.Atoi()`. The if statement would also hit if the resulting int is less than 1 since caddy doesn't support a `{label0}`. I don't think we should allow a value/placeholder that doesn't do anything and error instead. And I think for now we shouldn't diverge from the caddy implementation.

Reply:
That would fill the `err`, but wouldn't even result in a full error, just an empty replace.

Yeah I agree. We should return an error so that in both cases aka no int or {label0} would be handled by the fallback redirect code.